life_server/Functions/Systems/fn_handleBlastingCharge.sqf

```diff
@@ -4,12 +4,11 @@
     File Name: fn_handleBlastingCharge.sqf
     Information: Nothing really special, just handles the fed explosion over the server so if the client who place the charge logs out it still blows up.
 */
-private ["_bomb","_time"];
-_time = time + (5 * 60);
-waitUntil{(round(_time - time) < 1)};
+private _time = time + (5 * 60);
```

Comment:
Thinking about it - we can make this PR a bit better with giving a configurable bomb timer? It'd need edits in here, fn_demoChargeTimer and Config_Master.